### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by szpotona**
Move to 3.4
Remove hardcoded strings and defaults
Change a way of adding nodes to monitor
Add scaling groups
Add cleanup on healing delete

### CODE REVIEW

File: cloudify-dns-bind9-openstack-blueprint.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-tosca_definitions_version: cloudify_dsl_1_2
+tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
 
-  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml 
-  - http://www.getcloudify.org/spec/openstack-plugin/1.3.1/plugin.yaml 
-  - http://www.getcloudify.org/spec/diamond-plugin/1.3.1/plugin.yaml 
+  - http://www.getcloudify.org/spec/cloudify/3.4rc1/types.yaml
```
Comment (isaac-s) [inline cloudify-dns-bind9-openstack-blueprint.yaml]: @szpotona RC1 isn't released yet. Can you revert to 3.4m5?

Comment (szpotona) [inline cloudify-dns-bind9-openstack-blueprint.yaml]: I'm testing using rc1. I'd need to test it on 3.4m5 to be sure everything's working fine. Is this small delay acceptable?

Comment (isaac-s) [inline cloudify-dns-bind9-openstack-blueprint.yaml]: @szpotona yes. Go for it ASAP please.

Comment (szpotona) [inline cloudify-dns-bind9-openstack-blueprint.yaml]: @isaac-s Moved to 3.4m5 and tested both healing and Bind9

**Pull request merged**